platform/pulp/platform/models/progress.py

```diff
@@ -19,6 +19,19 @@
 _logger = logging.getLogger(__name__)
 
 
+def get_current_task_id():
```

Comment:
Cool. I'll change it back, but I won't install the ``pulp.server`` package. The tests will fail until your PR is merged, but the Python 3.y tests fail in any case (I've got a branch ready for that as well).